src/test/java/com/marginallyclever/makelangelo/plotter/settings/PlotterSettingsTest.java

```diff
@@ -41,6 +41,22 @@ void saveAndLoadConfig() {
         plotterSettings.setDrawFeedRate(21);
         plotterSettings.setAcceleration(22);
         plotterSettings.setHardwareName("TestRobot");
+        
+        int a = plotterSettings.getBlockBufferSize()+1;
```

Comment:
you should get the value but instead set the value. Why ? Because `blockBufferSize` and `segmentsPerSecond` could be initialized to `0` and thus `a` and `b` will be equals. In the assertion below, you won't catch that if the storing is buggy or not. Even if you know that `blockBufferSize` is, today, initialized at `16`. As you can set the value, you put a value different as the other in the test and in the assertion, you put a fixed value to compare it, not a calculated value based on the initialization of the object you are testing.
This is what is done for other attributes.

Reply:
I have no idea what you just said.  I make sure the value I set is never the default value, to be sure that it actually changed something.  If the default acceleration in PlotterSettings were already 22, the original tests would prove nothing.  My method works even if the default values change.